src/main/java/org/fundacionjala/coding/abel/CamelCase.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.abel;
+
+/**
+ *
+ */
+public final class CamelCase {
+    /**
+     *
+     */
+    protected CamelCase() {
+    }
+
+    /**
+     * @param str sting of imput
+     * @return value of return
+     */
+    public static String camelCase(final String str) {
+        StringBuffer contain = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)